insights/specs/datasources/dev.py

```diff
@@ -0,0 +1,34 @@
+"""
+Custom datasources to get the device names.
+"""
+
+from insights.core.context import HostContext
+from insights.core.exceptions import SkipComponent
+from insights.core.plugins import datasource
+from insights.parsers.blkid import BlockIDInfo
+from insights.combiners.virt_what import VirtWhat
+
+
+@datasource(BlockIDInfo, VirtWhat, HostContext)
```

Comment:
As the `BlockIDInfo` and `VirtWhat` are added as dependencies for this ds, please update the default_manifest for this dependent relationship as well
- https://github.com/RedHatInsights/insights-core/blob/master/insights/specs/manifests.py